fastapi/models/fastapi_endpoint.py

```diff
@@ -2,9 +2,10 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
 
 import logging
+from collections.abc import Awaitable, Callable
 from functools import partial
 from itertools import chain
-from typing import Any, Awaitable, Callable, Dict, List, Tuple, Type, Union
+from typing import Any
```

Comment:
Where these `Awaitable, Callable, Dict, List, Tuple, Type, Union` removed accidentally by ruff, or was it intentional?

Reply:
Hi, they were deprecated since version 3.9 ( i read the information here: https://docs.python.org/3/library/typing.html)
![image](https://github.com/OCA/rest-framework/assets/8724793/0bd7b555-7df8-4ddb-adf8-bcef5bce3ee2)

Reply:
You are right, thank you!